config/locales/simple_form.en.yml

```diff
@@ -42,7 +42,7 @@ en:
         debrief_date: "When your team will meet to debrief the findings of this iteration and shape subsequent plans - between 2-12 weeks from the start date."
         outcomes:
           title: "E.g. That busy parents have trouble finding/accessing meals their fussy eaters will happily consume."
-          success_criteria: "E.g. 8/10 busy parents cite this challenge."
+          success_criteria: "E.g. We have surveyed 100 busy parents and 8/10 cite this challenge."
```

Comment:
Surveying can be used to great effect in the right context but it's not typically the first step we encourage clients to take when they are looking to understand/validate a problem, so we might not want to highlight it in this example.